content-services/content-sources-backend
# Fixes 4307: add use_latest parameter to template

## Summary
- Adds the use_latest parameter to templates
- Adds validation on template update
- Adds validation on template create and update to prevent a date being set when use_latest is true

## Testing steps
1. Create a template with the use_latest parameter set to true
2. If you create with a date specified, and the parameter set to true, there should be a validation error.
3. Fetching that template, you should see the template set to true
4. Update the template to change the use_latest value should also work.
5. Validation now runs on template update, so you should not be able to update a template with a blank name, invalid arch, or invalid version.

## Checklist
